book/src/csi-driver-object.md

```diff
@@ -2,7 +2,7 @@
 
 ## Status
 
-Alpha
+Beta
```

Comment:
Add the following line underneath:

The `CSIDriver` object was introduced in In Kubernetes v1.12 as alpha. It was promoted to beta in Kubernetes 1.13.

Reply:
I know it's mentioned further down, but I think it would be convenient to also state the versions at the top

Reply:
+1 Please add this line here